torch_geometric/nn/models/schnet.py

```diff
@@ -268,7 +268,7 @@ def forward(self, z: Tensor, pos: Tensor,
                 a separate molecule with shape :obj:`[num_atoms]`.
                 (default: :obj:`None`)
         """
-        assert z.dim() == 1 and not z.dtype.is_floating_point
+        assert z.dim() == 1 and not z.dtype.is_floating_point()
```

Comment:
@rusty1s Did something change in torch?  I get this error when I try to invoke `is_floating_point` 
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [20], line 2
      1 x = torch.arange(10)
----> 2 x.dtype.is_floating_point()

TypeError: 'bool' object is not callable
```

I'm on torch 1.13 in case that makes a difference?

Reply:
I replaced it with `torch.is_floating_point(z)`. Hopefully, this will work across all versions.

Reply:
sounds good to me